_posts/2015-01-13-matryoshka-configurable-caching-library-for-php.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: with-comments
+title: "Matryoshka: A Configurable Caching Library for PHP"
+author: Marc Zych
+author_url: https://github.com/marczych
+summary: We recently open-sourced Matryoshka, a configurable caching library
+         for PHP which makes common operations easier and allows for on-the-fly
+         configuration.
+---
+
+Like most websites, we make heavy use of caching to reduce load on our servers and decrease page response times.
+Our caching daemon of choice is [memcached].
+The PHP extensions are certainly usable and provide all of the core functionality that you could need.
+However, we use a lot of patterns to make our day to day caching much easier that aren't provided by the extensions.
+
+In comes [Matryoshka], an open source caching library for PHP which makes common operations easier and allows for on-the-fly configuration.
```

Comment:
Maybe "So we developed..." instead of "In comes..."

I think we should take a bit more credit here, you wrote it from scratch.